### PR TITLE
Update registry.json

### PR DESCRIPTION
**Pull request by IvanLee113**
Added Echo Pool 
pool1wrt4jvd6nsxxtjws5lzl54zdjj8793u9jumcltqhsyqcgaa9d52
https://cardano-echo-pool.com/

### CODE REVIEW

File: registry.json [hunk 1]
```diff
@@ -1727,5 +1727,20 @@
       "telegram":"manizzler",
       "operatorName":"manizzler",
       "github":"manizzle"
+   },
+  {
+      "ticker":"ECHO",
+      "poolId":"pool1wrt4jvd6nsxxtjws5lzl54zdjj8793u9jumcltqhsyqcgaa9d52",
```
Comment (lophre) [inline registry.json:1733]: This is not a valid poolid (adapools id).  Your poolid is 70d75931ba9c0c65c9d0a7c5fa544d948fe2c78597378fac17810184

**Review by lophre — CHANGES_REQUESTED** [review]
Please update your poolid.

**Comment by IvanLee113** [comment]
Hello Lophre, 

There could be syntax error during the check, could you assist me? and also I have updated the pool ID, please advise. Thank you.

**Comment by lophre** [comment]
You have two successive curly braces... No problem, I will edit and merge.

**Review by lophre — APPROVED** [review]

**Pull request merged**